qutip/solve/stochastic.py

```diff
@@ -939,7 +939,7 @@ def _prespostdag(op):
 
     sso.sops = [[spre(op._cdc()) + spost(op._cdc()),
                  spre(op._cdc()),
-                 op.apply(_prespostdag)._f_norm2()] for op in sso.sc_ops]
+                 spre(op) * spost(op.dag())] for op in sso.sc_ops]
```

Comment:
could be `sprepost(op, op.dag())`?  Also, the `_f_norm2()` has gone.

Reply:
`apply`+`_f_norm2` do not create any of the cross terms coming from `sprepost`.
I removed the `_f_norm2` method.